plugins/adapter/discord/src/utils.ts

```diff
@@ -50,7 +50,7 @@ export function adaptMessage(meta: DC.Message, session: Partial<Session> = {}) {
   session.content = ''
   if (meta.content) {
     session.content = meta.content
-      .replace(/<@[!&](.+?)>/g, (_, id) => {
+      .replace(/<@!?(\d{17,19})>/g, (_, id) => {
```

Comment:
https://discord.com/developers/docs/reference#message-formatting

We still should handle role mentions with `&`. You can use `[!&]?` instead.

Reply:
Also sorry for taking so long to reply 😢